packages/router/__tests__/router-test.ts

```diff
@@ -1979,12 +1993,12 @@ describe("a router", () => {
         state: "idle",
         data: "FETCH 1",
       });
-      expect(shouldRevalidate.mock.calls.length).toBe(1);
-      expect(shouldRevalidate.mock.calls[0][0]).toMatchObject({
+      expect(shouldRevalidate.mock.calls.length).toBe(3);
+      expect(shouldRevalidate.mock.calls[2][0]).toMatchObject({
         currentParams: {},
-        currentUrl: new URL("http://localhost/fetch"),
+        currentUrl: new URL("http://localhost/"),
         nextParams: {},
-        nextUrl: new URL("http://localhost/fetch"),
+        nextUrl: new URL("http://localhost/child"),
```

Comment:
These now align with the navigation urls, not the identical `fetcher.load` url for both